### PR TITLE
build schema registry signed assemblies

### PR DESCRIPTION
**Pull request by mhowlett**
initial PR to test CI build of signed schema registry related assemblies. not yet for review

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -11,24 +11,28 @@ environment:
 
 build_script:
   - cmd: SET STRONG_NAME_OPTIONS=/p:SignAssembly=true;AssemblyOriginatorKeyFile=Confluent.Kafka.snk;PackageId=Confluent.Kafka.StrongName;Title=Confluent.Kafka.StrongName;AssemblyName=Confluent.Kafka.StrongName;BaseOutputPath=bin-sn\
+  - cmd: SET STRONG_NAME_OPTIONS_SR=/p:SignAssembly=true;AssemblyOriginatorKeyFile=Confluent.SchemaRegistry.snk;PackageId=Confluent.SchemaRegistry.StrongName;Title=Confluent.SchemaRegistry.StrongName;AssemblyName=Confluent.SchemaRegistry.StrongName;BaseOutputPath=bin-sn\
```
Comment (edenhill) [inline appveyor.yml]: what's with the trailing backslash?

Comment (mhowlett) [reply appveyor.yml]: i don't know, it's probably not necessary, this is just copying from the Confluent.Kafka case which works.

not for review. there's a lot more to be done here. this is for testing only.

**Comment by mhowlett** [comment]
following the advice here: https://www.pedrolamas.com/2018/09/11/start-strong-naming-your-assemblies/ , this PR introduces signing on all assemblies distributed via nuget - `Confluent.Kafka`, `Confluent.SchemaRegistry` and `Confluent.SchemaRegistry.Serdes`, and no longer builds `Confluent.Kafka.StrongName` (which is now irrelevant).

additionally, the associated unit test assemblies are signed since this acts as a test that all dependencies (including the above) are signed.

**Comment by mhowlett** [comment]
note also: https://github.com/confluentinc/avro/pull/16#discussion_r301192207

this pr resolves: https://github.com/confluentinc/confluent-kafka-dotnet/issues/973 - @longbaochenlong @zhoubo56 , feel free to confirm.

**Comment by mhowlett** [comment]
and @edenhill - ready for your OK.

File: appveyor.yml [hunk 2]
```diff
@@ -10,25 +10,27 @@ environment:
...
+
+  - cmd: IF "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION%)
+  - cmd: IF NOT "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION% --version-suffix ci-%APPVEYOR_BUILD_NUMBER%)
+  
   - cmd: IF "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.SchemaRegistry/Confluent.SchemaRegistry.csproj -c %CONFIGURATION%)
   - cmd: IF NOT "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.SchemaRegistry/Confluent.SchemaRegistry.csproj -c %CONFIGURATION% --version-suffix ci-%APPVEYOR_BUILD_NUMBER%)
-  - cmd: IF "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.SchemaRegistry.Serdes/Confluent.SchemaRegistry.Serdes.csproj -c %CONFIGURATION%)
-  - cmd: IF NOT "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.SchemaRegistry.Serdes/Confluent.SchemaRegistry.Serdes.csproj -c %CONFIGURATION% --version-suffix ci-%APPVEYOR_BUILD_NUMBER%)
+  
+  - cmd: IF "%APPVEYOR_REPO_TAG%" == "true" dotnet pack src/Confluent.SchemaRegistry.Serdes/Confluent.SchemaRegistry.Serdes.csproj -c %CONFIGURATION%
```
Comment (edenhill) [inline appveyor.yml]: Inconsistency: the IF statements above have () around the dotnet command

**Comment by mhowlett** [comment]
@longbaochenlong @zhoubo56 - the 1.2.0-RC1 packages are now on nuget and should all now be strong named. feel free to test this and confirm here. thanks!

**Pull request merged**